compliance/github/outside_collaborators/README.md

```diff
@@ -0,0 +1,33 @@
+## GitHub.com Outside Collaborators
+
+### What it does
+
+This Policy Template will get the outside collaborators for all repos within GitHub.com Org(s) and creates an incident if any do not match the whitelisted user accounts.
+
+
+### Parameters
+1. GitHub.com Organizations to check - Example: flexera
+2. Whitelisted Outside Collaborators - Example: flexera-ci
+3. Email address to send escalation emails to - Example: noreply@example.com
+
+
+### Policy Actions
```

Comment:
just curious - is it possible to have an action to remove non-permitted users?

Reply:
Yes it would be possible to add an action to remove the user's access to a repo.  For v1.0 hoping we can keep the scope to just reporting, but can add that as a TODO for this policy template